internal/command/balance.go

```diff
@@ -5,6 +5,10 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+// BalanceCommand Return account Balance
+// with the option to change to another currency
+// and change config file path
```

Comment:
"returns the current account balances with the option to change to another currency"